### ISSUE TITLE
Improve MediaSession info

### ISSUE THREAD

**Issue by Drafteed** [yuliskov/SmartTubeNext#1163]
Hello!

First, I want to say thank you for your work! 

I am a [Home Assistant](https://www.home-assistant.io/) user and use my TV with core [Google Cast integration](https://www.home-assistant.io/integrations/cast/). It uses information from the [MediaSession](https://developer.android.com/reference/android/media/session/MediaSession) to determine the current state. 

In general, everything is fine and works well, but the [Metadata](https://developer.android.com/reference/android/media/MediaMetadata) does not contain the duration of the video. From this, rewind cannot be used from the Home Assistant interface. Also `METADATA_KEY_ARTIST` contains the number of views and the upload date instead of the channel name (sometimes it may not be shown at all). `METADATA_KEY_ALBUM_ART` is also missing. All this works great with official YouTube client.

State from HA:
```yaml
volume_level: 0.07999999821186066
is_volume_muted: false
media_content_id: ''
media_duration: 0
media_position: 0.007
media_position_updated_at: '2021-12-24T14:27:40.937634+00:00'
media_title: 'Stephen Curry's best shots... with Google Photos'
app_id: AndroidNativeApp
app_name: STN Stable
entity_picture_local: null
friendly_name: Sony TV Cast
supported_features: 152511
```

I will be very grateful to you if you can fix this. Thank you again!

**Comment by yuliskov** [comment]
Hi. Is this bug fixed on the latest update 13.81?

**Comment by Drafteed** [comment]
Yes. Thank you! 🙏
